src/frontend/src/lib/components/ui/ButtonDone.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let type: 'submit' | 'reset' | 'button' = 'submit';
```

Comment:
Is it really ever used to submit a form? If not, should we also set it as `button` for all use case?

Reply:
I didn't want to change the logic to be honest, just in case it is needed as `submit` somewhere.

But it make sense that "Done" does not submit a form. 

I double checked and it is really never used in a form, you are right! Changed

Reply:
Makes sense. THx